neurovault/apps/main/static/css/style.css

```diff
@@ -211,3 +211,17 @@ ul.collection_nav li {
 .radio {
     padding-left:20px;
 }
+
+#fade {
+    display: none;
+    position:absolute;
+    top: 0%;
+    left: 0%;
```

Comment:
I'll probably be changing the color / spinner design entirely, I'm more interested in the functionality for this first look.